src/NServiceBus.Testing.Fakes/TestableInvokeHandlerContext.cs

```diff
@@ -29,6 +29,7 @@ public TestableInvokeHandlerContext(IMessageCreator messageCreator = null) : bas
         /// Moves the message being handled to the back of the list of available
         /// messages so it can be handled later.
         /// </summary>
+        [Obsolete("HandleCurrentMessageLater has been deprecated and will be removed in NServiceBus.Core Version 8.", true)]
```

Comment:
Shouldn't `IsError` be `false` since this will(likely) go out in a minor?

Reply:
@andreasohlund given that this already throws exception as of today, I think it's ok to make this an error and "violating" SemVer in that case?